run-cis-benchmark.sh

```diff
@@ -90,235 +123,680 @@ if [[ $(whoami) != "root" ]]; then
         echo "You must run this script as root"
         exit 1
 fi
-
 echo "CIS CentOS Linux 7 Benchmark (v2.2.0 - 12-27-2017)"
 echo
 echo hostname : `hostname`
 echo time     : `date`
 echo =================================================================================
+if [[ $(test_filter $1 1.1.1.1) == 0 ]]; then
```

Comment:
I would validate we're getting the expected type of input as $1, and then assign the value to a variable that describes what it is. Something like... 
```
if [[ $1 == "ValidInput1" || $1 == "ValidInput2" ]]; then
  testtype=$1
fi
```
Then you can call $(test_filter $testtype 1.1.1.1), which is a bit more self-documenting.